### PR TITLE
Use triple-slash comments for all interfaces

### LINKED ISSUE #473: Use triple-slash comments for documentation
Most of our documentation comments use `//`, which Xcode won't pick up. We should use `///` instead.

### PR DESCRIPTION
**Pull request by jspahrsummers**
_Depends on https://github.com/Mantle/Mantle/pull/474._

Resolves #473.

### CODE REVIEW

**Comment by robb** [comment]
Did you check if we still maintain the 80 char width limit?

File: Mantle/MTLModel+NSCoding.h [hunk 1]
```diff
@@ -8,121 +8,121 @@
...
+/// Determines how the +propertyKeys of the class are encoded into an archive.
+/// The values of this dictionary should be boxed MTLModelEncodingBehavior
+/// values.
+///
+/// Any keys not present in the dictionary will be excluded from the archive.
+///
+/// Subclasses overriding this method should combine their values with those of
+/// `super`.
+///
+/// Returns a dictionary mapping the receiver's +propertyKeys to default encoding
```
Comment (robb) [inline Mantle/MTLModel+NSCoding.h]: This line breaks the 80 char limit, there may be others.

**Comment by jspahrsummers** [comment]
> Did you check if we still maintain the 80 char width limit?

Nope! I don't really follow that as a hard-and-fast rule, and I'm definitely okay if it's over by one character.

**Comment by robb** [comment]
All cool then

**Comment by robrix** [comment]
> Nope! I don't really follow that as a hard-and-fast rule, and I'm definitely okay if it's over by one character.

Move fast and break 80-char limit rules.

**Pull request merged**